frontend/src/citizen-frontend/calendar/CalendarGridView.tsx

```diff
@@ -35,6 +37,7 @@ export default React.memo(function CalendarGridView({
   onCreateReservationClicked,
   onCreateAbsencesClicked,
   onReportHolidaysClicked,
+  holidayPeriods,
```

Comment:
Instead of passing this here, you could use the `useHolidayPeriods` hook for a bit less boilerplate.

Reply:
True, but then I would have a `Result<HolidayPeriod[]>` within the component tree where everything else is static data. This could (in theory) result in multiple renders, but I guess it's worth a try.